templates/index.html

```diff
@@ -44,57 +44,83 @@ <h1 class="display-3 custom-heading" >Empowering Women Software Devs</h1>
-                        <!--<div class="col-md-6">
-                        <img src="{% static 'djangogirlstz/images/Rectangle 14.webp' %}" class="img-fluid img-fluid-custom rounded-3" alt="Workshop Image">
-                        </div>-->
-                        <div class=" col-md-6 col-xm-12 col-sm-12 col-lg-6">
-                            <div class="image-container">
-                                <img src="{% static 'djangogirlstz/images/Rectangle 14.webp' %}" alt="First Image" class="top-image img-fluid">
-                                <img src="{% static 'djangogirlstz/images/Rectangle 14.webp' %}" alt="Second Image" class="bottom-image img-fluid">
+                        <!--<div class="blue-line"></div>-->
+                        <p class="mt-3">
+                            We are making technology more approachable by creating resources designed with empathy. During each of our events, 30-60 women build their first web application using HTML, CSS, Python, and Django.
+                        </p>
+                        <div class="row d-flex justify-content-between mt-4">
+                            <div class="col-xs-12 col-sm-12 col-md-4 col-lg-4 col-xl-4">
+                                <h2 id="counter-communities" class="large-font-bold">0</h2>
+                                <p class="">Communities</p>
                             </div>
-                        </div>
+                            <div class="col-xs-12 col-sm-12 col-md-4 col-lg-4 col-xl-4">
+                                <h2 id="counter-chapter-leads" class="large-font-bold">0</h2>
+                                <p class="">Chapter Leads</p>
+                            </div>
+                            <div class="col-xs-12 col-sm-12 col-md-4 col-lg-4 col-xl-4">
+                                <h2 id="counter-django-girls" class="large-font-bold">0</h2>
+                                <p class="">Django Girls Tanzania</p>
+                            </div>
+                        </div>   
+                    </div>
+                </div>
+                <!--<div class="col-md-6">
+                <img src="{% static 'djangogirlstz/images/Rectangle 14.webp' %}" class="img-fluid img-fluid-custom rounded-3" alt="Workshop Image">
+                </div>-->
+                <div class="col-md-6 col-xm-12 col-sm-12 col-lg-6">
+                    <div class="image-container">
+                        <img src="{% static 'djangogirlstz/images/Rectangle 14.webp' %}" alt="First Image" class="top-image img-fluid">
+                        <img src="{% static 'djangogirlstz/images/Rectangle 14.webp' %}" alt="Second Image" class="bottom-image img-fluid">
                     </div>
                 </div>
-            </div> 
-        </div>       
+            </div>
+        </div>
+    </div>       
+</div>
+
+<!-- jQuery and Counting Script -->
+<script src="https://code.jquery.com/jquery-3.6.0.min.js"></script>
```

Comment:
@deborachahe why are you including jquery here?
All import scripts should stay in the footer